### ISSUE TITLE
[BUG] [Linter] HTML col tag is always reported as orphan

### ISSUE THREAD

**Issue by dyve** [Riverside-Healthcare/djLint#140]
- [x] I'm on the [latest version](https://pypi.org/project/djlint/) of djLint
- [x] I've searched the [issues](https://github.com/Riverside-Healthcare/djLint/issues)
- [x] I've read the [docs](https://djlint.com)

## System Info
 - OS: macOS 12.0.1
 - Python Version: 3.8.12
 - djLint Version: 0.6.1

## Issue
Valid HTML `col` tag is always reported as orphan.

## How To Reproduce

Insert this valid HTML into a template:

```
<table>
  <colgroup>
    <col span="2" style="background-color:red">
    <col style="background-color:yellow">
  </colgroup>
  <tr>
    <th>ISBN</th>
    <th>Title</th>
    <th>Price</th>
  </tr>
  <tr>
    <td>3476896</td>
    <td>My first HTML</td>
    <td>$53</td>
  </tr>
</table>
```

Note that the `col` tags (inside the `colgroup` are reported as orphans.

## Configuration

```
# pyproject.toml
[tool.djlint]
# Reference: https://djlint.com/djlint/rules.html
profile = "django"
ignore = "H017,H019,T003,T002,H023,H014,H008,T028,H021,T001,H006,H013,H022"
blank_line_after_tag = "load,extends,include"
max_line_length = 120
```

**Comment by christopherpickering** [comment]
Thanks! If you add a closing `/` to them how is it reported? I supposed H017 should be triggered on the `col`.

```html
<col span="2" style="background-color:red" />
<col style="background-color:yellow" />
```

**Comment by christopherpickering** [comment]
Although it looks like I'm just missing `col` from the settings file for self closing tags... I think if its added there the rule will work correctly (not trigger in this case). https://github.com/Riverside-Healthcare/djLint/blob/85284228a25d38cf223d034fcc9926a6460b0051/src/djlint/settings.py#L678

I'm probably missing a few others. http://xahlee.info/js/html5_non-closing_tag.html

**Comment by dyve** [comment]
Actually `col` is a void tag and should not be closed

https://html.spec.whatwg.org/multipage/syntax.html#elements-2

**Comment by christopherpickering** [comment]
> Actually `col` is a void tag and should not be closed
> 
> https://html.spec.whatwg.org/multipage/syntax.html#elements-2

😁  Thats a big argument I'm gonna stay out of.

I pushed a fix for this. I will do a release in the next few days.
